ansible/roles/delfin-installer/scenarios/redis.yml

```diff
@@ -19,10 +19,10 @@
   register: redisservice
 
 - name: Ensure Redis is present
-  apt: 
-    name: redis-server 
-    state: latest
-  become: yes
+  shell: "{{ item }}"
+  with_items:
+    - ulimit -n 65536
```

Comment:
@joseph-v setting ulimit is fine, but what was the issue because of which this setting ulimit fixes it